backend/serializers/participant_serializer.py

```diff
@@ -16,8 +16,8 @@ class Meta:
         dump_only = ("id", "hacknights")
 
     first_name = fields.Str(required=True, validate=[validate.Length(max=50)])
-    last_name = fields.Str(validate=[validate.Length(max=50)])
-    email = fields.Email(validate=[validate.Length(max=200)])
-    github = fields.Str(validate=[validate.Length(max=200)])
+    last_name = fields.Str(required=True, validate=[validate.Length(max=50)])
+    email = fields.Email(required=True, validate=[validate.Length(max=200)])
+    github = fields.Str(required=True, validate=[validate.Length(max=200)])
     hacknights = fields.Nested("HacknightSchema", exclude=("participants",), many=True)
```

Comment:
I would consider validating minimum length of database not-nullable fields (first_name, last_name, github) to avoid adding participants with empty strings ("").

Reply:
I will create a separate ticket for adding regex validation for all fields.